server/Alumni/Controllers/UserController.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.AspNetCore.Mvc;
+using Alumni.Models;
+
+
+namespace Alumni.Controllers;
+
+[ApiController]
+[Route("api/users")]
+public class UserController : ControllerBase
+{
+    private readonly DataContext _context;
+    
+    public UserController(DataContext context)
+    {
+        _context = context;
+    }
+
+    [HttpPost("{id}/personal-info")]
+    public ActionResult<User> Edit(int id, [FromBody] EditPersonalInfoDTO dto)
+    {
+        var user = _context.Users.FirstOrDefault(
+            u => u.Id == id
+        );
+
+		if (user is null) {
+			return NotFound();
+		}
+
+        user.Name = dto.Name;
+		user.Login = dto.Email;
```

Comment:
You can format code using [csharpier](https://csharpier.com/)

Reply:
done